CHANGELOG.md

```diff
@@ -20,6 +20,24 @@
 
   ([PR #N](https://github.com/alphagov/digitalmarketplace-govuk-frontend/pull/N))
 
+- New component: Digital Marketplace Header component
+
+  *Requirement before using this component:*
+  - To use this the application must be using `govuk-frontend` template
```

Comment:
Will an app also need to have the phase info included somewhere before pulling in this component, as it's no longer in the header?

Reply:
Good point, thanks for raising it.